docs/spa/index.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: doc
+title: SPA integration guide
+abstract: How to integrate Single Page Webapps with DID.app
+---
+
+Single Page WebApps (SPAs) can use DID.app can authenticate users via OpenID Connect.
+Authenticating End Users takes two steps.
+
+1. Redirect a user to DID.app with an authentication request from your application.
+2. Fetch the user information from the authentication response.
+
+Fetching user information for authenticated users can be achieved via two methods.
+
+1. Have an API endpoint on your application backend fetch an id token.
+2. Use PKCE and fetch the id token from within the web app.
```

Comment:
```suggestion
2. Use PKCE (Proof Key for Code Exchange) and fetch the id token from within the web app.
```

Reply:
Have I not defined this earlier? If not it should be "Word Word (WW)" at first usage